scene/gui/color_picker.cpp

```diff
@@ -399,7 +399,7 @@ void ColorPicker::_screen_input(const InputEvent &ev)
 		Image img =r->get_screen_capture();
 		if (!img.empty())
 			last_capture=img;
-			r->queue_screen_capture();
```

Comment:
I think it's meant to be in the `if` block and Mariano just had used GDScript too much when writing it :p